fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraRepository.java

```diff
@@ -23,19 +23,4 @@
  */
 public interface FedoraRepository {
 
```

Comment:
Removed the login methods from FedoraRepository interface as the FedoraTransactionManager will be responsible for creating the FedoraTransaction. The FedoraRepository interface is now empty.

Reply:
I think that makes sense.  If we want to support logging into the OCFL client we can add that back in later.